src/test/shell/testenv.sh

```diff
@@ -19,12 +19,22 @@
 # TODO(bazel-team): This file is currently an append of the old testenv.sh and
 # test-setup.sh files. This must be cleaned up eventually.
 
-PLATFORM="$(uname -s | tr 'A-Z' 'a-z')"
-
-function is_windows() {
-  # On windows, the shell test is actually running on msys
-  [[ "${PLATFORM}" =~ msys_nt* ]]
-}
+PLATFORM="$(uname -s | tr [:upper:] [:lower:])"
+
+# # `uname` returns the current platform, e.g "MSYS_NT-10.0" or "Linux".
```

Comment:
Are those unnecessary anymore? Could we just remove them?

Reply:
Done.